package.json

```diff
@@ -14,67 +14,67 @@
 	},
 	"dependencies": {
 		"@next-auth/prisma-adapter": "^1.0.5",
-		"@next/eslint-plugin-next": "13.2.4-canary.0",
-		"@prisma/client": "^4.10.1",
-		"@radix-ui/react-dialog": "^1.0.2",
-		"@radix-ui/react-dropdown-menu": "^2.0.3",
-		"@radix-ui/react-popover": "^1.0.4",
-		"@radix-ui/react-tabs": "^1.0.2",
-		"@radix-ui/react-tooltip": "^1.0.4",
-		"@vercel/og": "^0.0.27",
+		"@next/eslint-plugin-next": "^13.2.4",
+		"@prisma/client": "^4.11.0",
+		"@radix-ui/react-dialog": "^1.0.3",
+		"@radix-ui/react-dropdown-menu": "^2.0.4",
+		"@radix-ui/react-popover": "^1.0.5",
+		"@radix-ui/react-tabs": "^1.0.3",
+		"@radix-ui/react-tooltip": "^1.0.5",
+		"@vercel/og": "^0.4.0",
 		"client-only": "^0.0.1",
 		"client-zip": "2.3.0",
-		"cmdk": "^0.1.22",
-		"jest": "^29.4.3",
+		"cmdk": "^0.2.0",
+		"jest": "^29.5.0",
 		"lodash.debounce": "^4.0.8",
-		"next": "13.2.4-canary.0",
-		"next-auth": "^4.19.2",
+		"next": "^13.2.4",
+		"next-auth": "^4.20.1",
 		"next-themes": "^0.2.1",
 		"react": "18.2.0",
-		"react-datepicker": "4.8.0",
+		"react-datepicker": "4.10.0",
 		"react-dom": "18.2.0",
 		"react-dropzone": "14.2.3",
 		"react-error-boundary": "^3.1.4",
 		"react-feather": "^2.0.10",
 		"react-hot-toast": "2.4.0",
 		"server-only": "^0.0.1",
-		"swr": "^2.0.4",
+		"swr": "^2.1.0",
 		"textarea-markdown-editor": "1.0.4",
 		"ts-jest": "^29.0.5",
 		"uuid": "^9.0.0",
 		"zlib": "^1.0.5"
 	},
 	"devDependencies": {
-		"@next/bundle-analyzer": "13.1.7-canary.26",
-		"@total-typescript/ts-reset": "^0.3.7",
+		"@next/bundle-analyzer": "13.2.4",
+		"@total-typescript/ts-reset": "^0.4.2",
 		"@types/bcrypt": "^5.0.0",
 		"@types/git-http-backend": "^1.0.1",
-		"@types/jest": "^29.4.0",
+		"@types/jest": "^29.4.1",
 		"@types/lodash.debounce": "^4.0.7",
-		"@types/node": "18.11.18",
-		"@types/react": "18.0.27",
-		"@types/react-datepicker": "4.8.0",
-		"@types/react-dom": "18.0.10",
+		"@types/node": "18.15.3",
+		"@types/react": "18.0.28",
+		"@types/react-datepicker": "4.10.0",
+		"@types/react-dom": "18.0.11",
 		"@types/uuid": "^9.0.1",
-		"@typescript-eslint/eslint-plugin": "^5.53.0",
-		"@typescript-eslint/parser": "^5.53.0",
+		"@typescript-eslint/eslint-plugin": "^5.55.0",
+		"@typescript-eslint/parser": "^5.55.0",
 		"@wcj/markdown-to-html": "^2.2.1",
 		"clsx": "^1.2.1",
 		"cross-env": "7.0.3",
 		"csstype": "^3.1.1",
 		"dotenv": "^16.0.3",
-		"eslint": "8.33.0",
-		"eslint-config-next": "13.1.7-canary.26",
-		"jest-mock-extended": "^3.0.2",
+		"eslint": "8.36.0",
+		"eslint-config-next": "13.2.4",
+		"jest-mock-extended": "^3.0.3",
 		"next-unused": "0.0.6",
 		"postcss-flexbugs-fixes": "^5.0.2",
 		"postcss-hover-media-feature": "^1.0.2",
 		"postcss-nested": "^6.0.1",
 		"postcss-preset-env": "^8.0.1",
-		"prettier": "2.8.3",
-		"prisma": "^4.10.1",
-		"typescript": "4.9.4",
-		"typescript-plugin-css-modules": "4.1.1"
+		"prettier": "2.8.4",
+		"prisma": "^4.11.0",
+		"typescript": "4.9.5",
+		"typescript-plugin-css-modules": "4.2.3"
```

Comment:
Ok for this PR, but in the future can you include dependency updates in a separate PR / commit?

Reply:
Yes sure :+1: